GoogleForCreators/web-stories-wp
# Display correct panels for page without background

## Summary

Removes Overlay and Style Panel from empty background, as in Figma.


## Relevant Technical Choices
N/A


## To-do

Would be good to add a Karma test later to test that the Panels are displayed correctly per each element / Video background / Image background / Empty background


## User-facing changes

Page without background will now only display Style Presets Panel


## Testing Instructions

I
- Add a new Page
- Verify that the new Page displays only Style Presets & Page panel where the background color can be selected.
- Add background Media
- Verify that the relevant panels will appear in addition to the Presets and Page panel: Overlay, Accessibility, Size & Position


II
1. Add a new Page
2. Assign Page color
3. Verify that the correct color appears in the front-end as well.
---

For reference:
<img width="667" alt="Screenshot 2020-06-04 at 10 41 44" src="https://user-images.githubusercontent.com/3294597/83734993-0635b900-a650-11ea-82fc-5a17f08a912b.png">



Fixes #1177
